android/java/org/chromium/chrome/browser/externalnav/BraveExternalNavigationHandler.java

```diff
@@ -29,18 +28,9 @@ public BraveExternalNavigationHandler(ExternalNavigationDelegate delegate) {
 
     @Override
     public OverrideUrlLoadingResult shouldOverrideUrlLoading(ExternalNavigationParams params) {
```

Comment:
this override is no longer needed, right?

Reply:
we need it for the other usecase we have where we need to redirect users to native page if they enter brave://adblock. currently that part is commented out.